ext/page_log/palm/palm.c

```diff
@@ -391,6 +467,30 @@ palm_init_context(PALM *palm, PALM_KV_CONTEXT *context)
     context->last_materialized_lsn = palm->last_materialized_lsn;
 }
 
+/*
+ * palm_init_lsn --
+ *     Remember the current LSN when we started PALM.
+ */
+static int
+palm_init_lsn(PALM *palm)
+{
+    PALM_KV_CONTEXT context;
+    int ret;
+
+    palm_init_context(palm, &context);
+    PALM_KV_RET(palm, NULL, palm_kv_begin_transaction(&context, palm->kv_env, true));
+
+    /*
+     * Get the LSN. If it's never been set, we'll get not found, but that's okay, that will leave
+     * out beginning LSN at zero, which is fine for our purposes.
```

Comment:
Should this be "leave *our* beginning LSN", or have I misunderstood the comment?

Reply:
Yes.  Fixed in 4dab1a3e8c